pkg/oci/oci.go

```diff
@@ -98,12 +97,9 @@ const (
 
 // getLocalOciStore returns a single local oci store. oci.Store is concurrently safe only
```

Comment:
This is not `a single local oci` store anymore. We are creating new ones every time in this PR